packages/unit-vitest/package.json

```diff
@@ -39,18 +39,18 @@
   },
   "devDependencies": {
     "@types/lodash-es": "^4.17.12",
-    "@vitest/ui": "^2.0.5",
-    "@vue/test-utils": "^2.4.4",
+    "@vitest/ui": ">=1.6.1 <2.0.0 || >=2.1.9 <3.0.0 || >=3.0.5",
+    "@vue/test-utils": "^2.0.5",
```

Comment:
```suggestion
    "@vue/test-utils": "^2.4.6",
```

Reply:
This still holds

Reply:
Ops, my bad. This wasn't intentional. I thought to have applied the fix together with the others. Fixed.